src/app/get-started/custom-workspace-tab/temporary-storage-row/temporary-storage-row.controller.ts

```diff
@@ -26,58 +28,81 @@ export class TemporaryStorageRowController implements ng.IController, ITemporary
   ];
 
   // component bindings
-  temporary?: boolean;
-  onChange: (eventObj: { $temporary: boolean, $default: boolean }) => void;
+  storageType?: string
+  onChangeStorageType: (eventObj: { $storageType: string; $default: boolean; }) => void;
+  storageDescription?: string
   // init promise
   initPromise: ng.IPromise<string>;
   // template fields
   temporaryStorageSwitch: IChePfSwitchProperties;
+  storageSelect: IChePfSelectProperties<string>;
   // injected services
   private cheWorkspace: CheWorkspace;
 
   constructor(cheWorkspace: CheWorkspace) {
     this.cheWorkspace = cheWorkspace;
-
-    this.temporaryStorageSwitch = {
+    this.storageSelect = {
       config: {
-        name: 'temporaryStorage',
+        items: [STORAGE_TYPE.PERSISTENT.label, STORAGE_TYPE.EPHEMERAL.label, STORAGE_TYPE.ASYNCHRONOUS.label],
+        placeholder: 'Select a storage template'
+      },
+      value: STORAGE_TYPE.EPHEMERAL.label,
+      onSelect: value => {
+        console.log(value)
```

Comment:
Do we really need it ```  console.log(value) ```?

Reply:
I like the old one ```onChange```

In the case with ```onSelect```, it does not clear what will be if I select but don't change the value.
WDYT?

Reply:
in the case with ```onSelect``` and the same value, it shouldn't call  ```this.onChangeStorageType```
https://github.com/eclipse/che-dashboard/pull/58/files#diff-fbf64ab21e61829eed60e7ce864f9cb0R106